cmd/clusterctl/cmd/describe_cluster.go

```diff
@@ -161,9 +162,8 @@ func runDescribeCluster(name string) error {
 // printObjectTree prints the cluster status to stdout.
 func printObjectTree(tree *tree.ObjectTree) {
 	// Creates the output table
-	tbl := uitable.New()
-	tbl.Separator = "  "
-	tbl.AddRow("NAME", "READY", "SEVERITY", "REASON", "SINCE", "MESSAGE")
+	tbl := tablewriter.NewWriter(os.Stdout)
+	tbl.SetHeader([]string{"NAME", "READY", "SEVERITY", "REASON", "SINCE", "MESSAGE"})
```

Comment:
We want to retain the look of the describe cluster commands output. Therefore, I think we set some formatting rules on this new table to match the old look. I believe the default formatting rules of the new table library are different from the old library.

New look: 
```
❯ clusterctl-dev describe cluster first-cluster                      
+-----------------------------------------------------+-------+----------+--------+-------+---------+
|                        NAME                         | READY | SEVERITY | REASON | SINCE | MESSAGE |
+-----------------------------------------------------+-------+----------+--------+-------+---------+
| Cluster/first-cluster                               | True  |          |        | 7m5s  |         |
| ├─ClusterInfrastructure -                           | True  |          |        | 8m4s  |         |
| DockerCluster/first-cluster-qkhqv                   |       |          |        |       |         |
| ├─ControlPlane -                                    | True  |          |        | 7m5s  |         |
| KubeadmControlPlane/first-cluster-6gt8h             |       |          |        |       |         |
| │                                                   | True  |          |        | 7m7s  |         |
| └─Machine/first-cluster-6gt8h-cms7r                 |       |          |        |       |         |
| └─Workers                                           |       |          |        |       |         |
|                                                     | True  |          |        | 5m30s |         |
| ├─MachineDeployment/first-cluster-md-0-5mtp5        |       |          |        |       |         |
|   │                                                 | True  |          |        | 6m30s |         |
| └─Machine/first-cluster-md-0-5mtp5-d5f5b6d59-7kcx9  |       |          |        |       |         |
|                                                     | True  |          |        | 6m    |         |
| └─MachineDeployment/first-cluster-md-1-5nwrf        |       |          |        |       |         |
|                                                     | True  |          |        | 6m30s |         |
| └─Machine/first-cluster-md-1-5nwrf-7864c9696d-5hgwm |       |          |        |       |         |
+-----------------------------------------------------+-------+----------+--------+-------+---------+
```

Expected look:
```
❯ clusterctl describe cluster first-cluster  
NAME                                                         READY  SEVERITY  REASON  SINCE  MESSAGE
Cluster/first-cluster                                        True                     8m5s          
├─ClusterInfrastructure - DockerCluster/first-cluster-qkhqv  True                     9m4s          
├─ControlPlane - KubeadmControlPlane/first-cluster-6gt8h     True                     8m5s          
│ └─Machine/first-cluster-6gt8h-cms7r                        True                     8m7s          
└─Workers                                                                                           
  ├─MachineDeployment/first-cluster-md-0-5mtp5               True                     6m30s         
  │ └─Machine/first-cluster-md-0-5mtp5-d5f5b6d59-7kcx9       True                     7m30s         
  └─MachineDeployment/first-cluster-md-1-5nwrf               True                     7m            
    └─Machine/first-cluster-md-1-5nwrf-7864c9696d-5hgwm      True                     7m30s         
```

A note: I had to replace l.172 with `tbl.Render()` to get the above output. The current code prints raw output. The new table writer needs a call to the `Render` function to print the formatted output. Please fix that.